dfx/index.ts

```diff
@@ -92,3 +127,48 @@ export async function getIdentity(
     );
     return Secp256k1KeyIdentity.fromPem(await readFile(identityPath, 'utf-8'));
 }
+
+export function getPemKey(identityName: string = getIdentityName()): string {
+    console.info();
+    console.info(`Exporting PEM key for ${identityName}`);
+    console.info(`You may be prompted for ${identityName}'s password`);
+    console.info();
+    const cmd = `dfx identity export ${identityName}`;
+    const result = execSync(cmd, {
+        stdio: ['inherit', 'pipe', 'inherit'] // TODO I would prefer it to pipe the stderr but it will fail immediately if you do that
+    })
+        .toString()
+        .trim();
+    return result;
+}
+
+export function getIdentity(identityName?: string): Secp256k1KeyIdentity {
```

Comment:
We should probably rename this to getSecp256k1Identity, or allow passing in the type of identity, since this won't work generally